### PR TITLE
Left + depthmap --> right view

### PR DESCRIPTION
**Pull request by annikabrundyn**

### CODE REVIEW

**Comment by annikabrundyn** [comment]
mm i need to add a test but it requires providing a model checkpoint... thinking about how to do this

**Comment by annikabrundyn** [comment]
> mm i need to add a test but it requires providing a model checkpoint... thinking about how to do this

@jeswan I've added a test but it requires manually giving a checkpoint path - its a good sanity check for me for now but will think about how to generalize the test another time

File: tests/models/test_dm_right_model.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import pytest
+import pytorch_lightning as pl
+
+from models.right_view.dm_right_model import DepthMapRightModel
+from data.right_data import RightDaVinciDataModule
+from models.callbacks.right_dm_callback import RightDepthMapCallback
+
+
+# TODO: dont know how to make this cleaner
+DEPTH_MAP_CHECKPOINT_PATH = "/Users/annikabrundyn/Developer/da_vinci/grid_artifacts/final_dm_artifacts/grid_artifacts/m10-2drop0-color-exp0/version_0/checkpoints/epoch=155.ckpt"
```
Comment (jeswan) [inline tests/models/test_dm_right_model.py:10]: if you want, you can parameterize this just like `data_dir` below. up to you!

File: src/models/right_view/dm_right_model.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        self.right_model = RightUNet(num_classes=self.num_classes,
+                                     input_channels=self.input_channels,
+                                     num_layers=self.hparams.num_layers,
+                                     features_start=self.hparams.features_start,
+                                     bilinear=self.hparams.bilinear)
+
+    def forward(self, x):
+
+        # depth map model
+        reshaped_dm, pred_dm = self.trained_depth_model(x)
```
Comment (jeswan) [inline src/models/right_view/dm_right_model.py:64]: Why are there two outputs?

**Review by jeswan — APPROVED** [review]

**Pull request merged**